tests/run-make/raw-dylib-cross-compilation/lib.rs

```diff
@@ -8,7 +11,7 @@
 trait Sized {}
 
 #[link(name = "extern_1", kind = "raw-dylib")]
-extern {
+extern "C" {
```

Comment:
And the extraneous formatting-related changes should be removed from this and the other files below.